extensions/sockets/src/main/java/org/mule/module/socket/internal/DefaultUdpSocketProperties.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.socket.internal;
+
+import org.mule.module.socket.api.udp.UdpSocketProperties;
+
+/**
+ * Default mutable implementation of the {@code UdpSocketProperties} interface.
+ *
+ * @since 4.0
+ */
+public class DefaultUdpSocketProperties extends AbstractUdpSocketProperties implements UdpSocketProperties
```

Comment:
is there an immutable implementation? otherwise the Abstract class is useless and just move all that stuff here